star/test_suite/custom_colors/.gitignore

```diff
@@ -0,0 +1,28 @@
+# Ignore the data directory
```

Comment:
I think this should be rationalised against the top-level `.gitignore`, and the fact that we shouldn't encounter some of these files (e.g. `.dll`).

The only other two test cases that have `.gitignore` files are `ccsn_IIp` and `tzo`, and those are mostly for a few very specific model files.